llvm/lib/Target/AMDGPU/GCNRegPressure.cpp

```diff
@@ -110,7 +110,7 @@ bool GCNRegPressure::less(const MachineFunction &MF, const GCNRegPressure &O,
   if (Occ != OtherOcc)
     return Occ > OtherOcc;
 
-  unsigned MaxVGPRs = ST.getMaxNumVGPRs(MF);
+  unsigned MaxVGPRs = ST.getMaxNumVGPRs(MF, /*WholeRegisterFile*/ true);
```

Comment:
```suggestion
  unsigned MaxVGPRs = ST.getMaxNumVGPRs(MF, /*WholeRegisterFile=*/ true);
```

Parameter name comments need a trailing = for clang-format to recognize them